packages/fuels-core/src/code_gen/functions_gen.rs

```diff
@@ -741,32 +734,6 @@ pub fn hello_world(
         assert_eq!(result.unwrap().to_string(), ":: std :: vec :: Vec < u64 >");
     }
     #[test]
-    fn test_expand_input_param_custom_type() {
-        let def = Function::default();
-        let struct_type = ParamType::Struct(vec![ParamType::Bool, ParamType::U64]);
-        let struct_prop = Property {
-            name: String::from("unused"),
-            type_field: String::from("struct babies"),
-            components: None,
-        };
-        let struct_name = Some(&struct_prop);
-        let result = expand_input_param(&def, "unused", &struct_type, &struct_name);
-        // Notice the removed plural!
-        assert_eq!(result.unwrap().to_string(), "Baby");
-
-        let enum_type =
-            ParamType::Enum(EnumVariants::new(vec![ParamType::U8, ParamType::U32]).unwrap());
-        let enum_prop = Property {
-            name: String::from("unused"),
-            type_field: String::from("enum babies"),
-            components: None,
-        };
-        let enum_name = Some(&enum_prop);
-        let result = expand_input_param(&def, "unused", &enum_type, &enum_name);
-        // Notice the removed plural!
-        assert_eq!(result.unwrap().to_string(), "Baby");
-    }
-    #[test]
```

Comment:
Why did you remove the #[test] (macro?) from test_expand_input_param_struct_wrong_name()?

Reply:
Uh... weird diff. It's still there, see whole file: https://github.com/FuelLabs/fuels-rs/blob/e07c117d2de67f66a1e67c3959b0d1834b1e6f3f/packages/fuels-core/src/code_gen/functions_gen.rs#L736